.coveragerc

```diff
@@ -12,5 +12,5 @@ omit =
 [report]
 show_missing = True
 precision = 2
-fail_under = 97.13
+fail_under = 97.12
```

Comment:
I think we removed so much code here that the untested portions of H now make up a bigger percentage.